### PR TITLE
Fix img attributes

### PR DESCRIPTION
**Pull request by rowbean**
added empty alt attributes

### CODE REVIEW

File: themes/codefor-theme/layouts/shortcodes/ziele/accordion-goals.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
 							</div>
 							<div class="col d-flex align-items-start justify-content-center justify-content-sm-end">
 								<button class="collapsed btn btn-close" type="button" data-toggle="collapse" data-target="#collapse-{{ $index }}" aria-expanded="true" aria-controls="collapse-{{ $index }}">
-									<img class="img-fluid" src="/img/close-icon.svg">
+									<img class="img-fluid" src="/img/close-icon.svg" alt="">
```
Comment (wagnerpeer) [inline themes/codefor-theme/layouts/shortcodes/ziele/accordion-goals.html:23]: Why are we using an empty alternative description? I would like to see a real description like "Closing Icon 'X'"

Comment (ubergesundheit) [inline themes/codefor-theme/layouts/shortcodes/ziele/accordion-goals.html:23]:
```suggestion
									<img class="img-fluid" src="/img/close-icon.svg" aria-hidden="true">
```

File: themes/codefor-theme/layouts/shortcodes/ziele/accordion-goals.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
 				<div class="row justify-content-center">
 					<div class="col-9 col-sm-5 col-xl-4 d-flex flex-column align-items-center">
 						<div class="px-md-4 w-100">
-							<img class="img-fluid p-2" src="/icons/{{ $element.icon }}">
+							<img class="img-fluid p-2" src="/icons/{{ $element.icon }}" alt="">
```
Comment (ubergesundheit) [inline themes/codefor-theme/layouts/shortcodes/ziele/accordion-goals.html:10]:
```suggestion
							<img class="img-fluid p-2" src="/icons/{{ $element.icon }}" aria-hidden="true">
```

Comment (wagnerpeer) [inline themes/codefor-theme/layouts/shortcodes/ziele/accordion-goals.html:10]: I know, that the icons are picked, based on short codes. However, at least the name of the icon could give a hint if used as alt text.
```html
<img class="img-fluid p-2" src="/icons/{{ $element.icon }}" alt="{{ $element.icon }}">
```

**Review by wagnerpeer — CHANGES_REQUESTED** [review]
I like the idea of adding alt tags. But please try to add descriptive text as intended by using such tags.

**Comment by rowbean** [comment]
> 
> 
> I like the idea of adding alt tags. But please try to add descriptive text as intended by using such tags.

I like using alt tags too, but in this case i don't see a content value - https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML#Empty_alt_attributes

**Review by wagnerpeer — APPROVED** [review]

**Comment by iamsilvio** [comment]
Stale PR 
empty alt tags are not helpful

**Pull request closed without merging**